scripts/kollect.sh

```diff
@@ -77,7 +77,7 @@ done
 
 echo '"$@"'
 
-if [[ -z "$kollected" ]]
+if [[ -z "${kollected:-}" ]]
 then
     echo >&2 "Did not collect any Kore files!"
     exit 1
```

Comment:
This is what we do when it fails. We should add an `else` branch where we remove the temporary directory. We will need to add a variable where we save the name of the temporary directory.